lib/Common/DataStructures/FixedBitVector.h

```diff
@@ -242,18 +242,19 @@ void BVFixed::SetRange(Container* value, BVIndex start, BVIndex len)
     BVIndex oStart = BVUnit::Offset(start);
     BVIndex oEnd = BVUnit::Offset(end);
 
-    BVUnit::BVUnitTContainer temp;
-    BVUnit::BVUnitTContainer* bits;
+    typedef typename std::conditional<sizeof(Container) == 1,uint8_t,BVUnit::BVUnitTContainer>::type operandtype;
```

Comment:
Fair point; I'll just mark this whole function as NO_SANITIZE_ADDRESS instead, and that should make the changeset significantly simpler too.